src/components/subject-level-chips/SubjectLevelChips.tsx

```diff
@@ -1,28 +1,33 @@
 import { FC, useMemo } from 'react'
+import { useTranslation } from 'react-i18next'
 
 import Box from '@mui/material/Box'
-import { SxProps } from '@mui/material'
+import Typography from '@mui/material/Typography'
 
 import AppChip from '~/components/app-chip/AppChip'
 import palette from '~/styles/app-theme/app.pallete'
 import { spliceSx } from '~/utils/helper-functions'
 
-import { ProficiencyLevelEnum } from '~/types'
+import { ProficiencyLevelEnum, SubjectLevelChipsSx } from '~/types'
 import { styles } from '~/components/subject-level-chips/SubjectLevelChips.styles'
 
 interface SubjectLevelChipsProps {
-  sx?: SxProps
+  cardSquareStyle: boolean
+  sx?: SubjectLevelChipsSx
   proficiencyLevel: ProficiencyLevelEnum | ProficiencyLevelEnum[]
   subject: string
   color?: string
 }
 
 const SubjectLevelChips: FC<SubjectLevelChipsProps> = ({
+  cardSquareStyle = false,
```

Comment:
poor naming. and do a separate component for this, talk with @tsivkadi about it. she does the same

Reply:
> poor naming. and do a separate component for this, talk with @tsivkadi about it. she does the same

fixed